packages/react-components/react-table/stories/DataGrid/KeyboardInteraction.stories.tsx

```diff
@@ -0,0 +1,233 @@
+      return <TableCellLayout truncate>{item.lastUpdated.label}</TableCellLayout>;
+    },
+  }),
+  createTableColumn<Item>({
+    columnId: 'lastUpdate',
+    compare: (a, b) => {
+      return a.lastUpdate.label.localeCompare(b.lastUpdate.label);
+    },
+    renderHeaderCell: () => {
+      return 'Last update';
+    },
+    renderCell: item => {
+      return (
+        <TableCellLayout truncate media={item.lastUpdate.icon}>
+          {item.lastUpdate.label}
+        </TableCellLayout>
+      );
+    },
+  }),
+];
+
+export const KeyboardInteraction = () => {
+  return (
+    <DataGrid
+      items={items}
+      columns={columns}
+      sortable
+      getRowId={item => item.file.label}
+      resizableColumns
+      columnSizingOptions={{
+        file: {
+          minWidth: 80,
+          defaultWidth: 120,
+        },
+        author: {
+          defaultWidth: 180,
+          minWidth: 120,
+          idealWidth: 180,
+        },
+      }}
+    >
+      <DataGridHeader>
+        <DataGridRow selectionCell={{ 'aria-label': 'Select all rows' }}>
+          {({ renderHeaderCell, columnId }, state) => (
+            <Menu openOnContext>
+              <MenuTrigger>
+                <DataGridHeaderCell>{renderHeaderCell()}</DataGridHeaderCell>
+              </MenuTrigger>
+              <MenuPopover>
+                <MenuList>
+                  {state.accessibilityMenuItems?.map(item => (
+                    <MenuItem key={item.key} onClick={item.getClickHandler(columnId)}>
+                      {item.label}
+                    </MenuItem>
+                  ))}
+                </MenuList>
+              </MenuPopover>
+            </Menu>
+          )}
+        </DataGridRow>
+      </DataGridHeader>
+      <DataGridBody<Item>>
+        {({ item, rowId }) => (
+          <DataGridRow<Item> key={rowId} selectionCell={{ 'aria-label': 'Select row' }}>
+            {({ renderCell }) => <DataGridCell>{renderCell(item)}</DataGridCell>}
+          </DataGridRow>
+        )}
+      </DataGridBody>
+    </DataGrid>
+  );
+};
+
+KeyboardInteraction.storyName = 'Keyboard Interaction (preview)';
+KeyboardInteraction.parameters = {
+  docs: {
+    description: {
+      story: [
+        'For accessibility, the `DataGrid` component supports keyboard navigation and screen reader navigation',
+        'To make features like column resizing work with keyboard navigation, the `Menu` component is used to provide',
+        ' a context menu for the header cells, which allows the user to access other DataGrid features.',
+        '',
+        'Once an option is selected, the arrows keys can be used to interact with the `DataGrid` (e.g. resize columns).',
+        '`ESC`, `ENTER`, or `SPACE` can be used to return to the original navigation mode.',
+        '',
+        'The `DataGrid` component is a WIP component and is not yet fully accessible. We are working on improving',
+        'the accessibility of this component and welcome any feedback.',
```

Comment:
```suggestion
```

The `DataGrid` is fully accessible without the column resizing feature which is preview. All other features that are stable should be fully accessible

Reply:
I will remove this then